model.py

```diff
@@ -1885,8 +1885,15 @@ def author_contributor_tiers(cls):
 
     @classmethod
     def lookup(cls, _db, name=None, viaf=None, lc=None, aliases=None,
-               extra=None):
-        """Find or create a record for the given Contributor."""
+               extra=None, create_new=True):
```

Comment:
When I search for contributor equivalencies, I do:
- get contributor from our db
- look them up in viaf
- for each viaf-returned cluster that is deemed the same person as looked-up contributor, I see if that cluster is in our db.  If it is, then I label it as related person.  So I don't want that lookup to create a new contributor here.